Developer-Guide.md

```diff
@@ -273,22 +291,199 @@ $ popd
+# is "trusted".
+# Containers can be run through different container runtimes, depending on
+# the trust hints we receive from kubelet:
+# - If kubelet tags a container workload as untrusted, crio will try first to
+# run it through the untrusted container workload runtime. If it is not set,
+# crio will use the trusted runtime.
+# - If kubelet does not provide any information about the container workload trust
+# level, the selected runtime will depend on the default_container_trust setting.
+# If it is set to "untrusted", then all containers except for the host privileged
+# ones, will be run by the runtime_untrusted_workload runtime. Host privileged
+# containers are by definition trusted and will always use the trusted container
+# runtime. If default_container_trust is set to "trusted", crio will use the trusted
+# container runtime for all containers.
+default_workload_trust = "untrusted"
+
+```
+
+Restart CRI-O to take changes into account
+```
+$ sudo systemctl restart crio
+```
+
+### CRI-containerd
+
+If you picked CRI-containerd, please follow the instructions
+[here](https://github.com/containerd/cri#getting-started-for-developers)
+to get it properly installed.
+
+And for the customization part so that CRI-containerd will pick Kata Containers
+runtime, please follow our internal documentation
+[here](https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#configure-containerd-to-use-kata-containers).
+
+## Install Kubernetes
+
+Depending on what your needs are and what you expect to do with Kubernetes,
+please refer to the following
+[documentation](https://kubernetes.io/docs/setup/pick-right-solution/) to
+install it correctly.
+
+Kubernetes talks with CRI implementations through a `container-runtime-endpoint`,
+also called CRI socket. This socket path is different depending on which CRI
+implementation you choosed, and the kubelet service has to be updated accordingly.
+
+### Configure for CRI-O
+
+_/etc/systemd/system/kubelet.service.d/0-crio.conf_
+```
+[Service]                                                 
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///var/run/crio/crio.sock"
+```
+
+### Configure for CRI-containerd
+
+_/etc/systemd/system/kubelet.service.d/0-cri-containerd.conf_
+```
+[Service]                                                 
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
+```
+For more details about CRI-containerd, click
+[here](https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#configure-kubelet-to-use-containerd).
+
+## Run a Kubernetes pod with Kata Containers
+
+After you have updated your kubelet service based on the CRI implementation you
```

Comment:
Lines 451-452 suggested rewrite: After you update your kubelet service based on the CRI implementation you are using, reload and restart kubelet. Then, start your cluster:

Reply:
done